docs/applications/containers/how-to-deploy-the-elastic-stack-on-kubernetes/index.md

```diff
@@ -44,13 +44,16 @@ If you remove the resources afterward, you will only be billed for the hour(s) t
 
 ## Before You Begin
 
+{{< note >}}
+This guide uses Kubernetes services which are private by default. Local listeners are opened which allow you to access the services on your local browser, however, web servers and NodeBalancers are out scope for this guide. Due to this, you should complete the steps of this guide from your local computer or from a computer that you will give you access the web browser. If you wish to be able to access these services from a public domain, please see our guide on [Getting Started with NodeBalancers](/docs/platform/nodebalancer/getting-started-with-nodebalancers/).
+{{< /note >}}
+
```

Comment:
Added this note about the local computer, Kubernetes private services, and access via node balancer etc. Let me know if you think this is appropriately worded.

Reply:
Great, thank you!